spec/helpers/movies_helper_spec.rb

```diff
@@ -14,7 +14,11 @@
 
     it 'returns an image tag if the movie has a poster path' do
       allow(movie).to receive(:poster_path).and_return('tester')
-      expect(helper.image_for(movie)).to eq(image_tag("http://image.tmdb.org/t/p/w185#{movie.poster_path}"))
+      expect(helper.image_for(movie)).to eq(
+        image_tag("http://image.tmdb.org/t/p/w185#{movie.poster_path}",
+          title: movie.title,
+          alt: movie.title)
+        )
```

Comment:
Ah, that makes sense. Super weird that Travis didn't give us a clear heads-up on that.